aem-authoring-toolkit-plugin/src/test/com/exadel/aem/toolkit/core/util/writer/TestXmlWriterHelper.java

```diff
@@ -90,6 +100,25 @@ public static boolean doTest(String testedClass, Path pathToExpectedFiles) throw
         return actualFiles;
     }
 
+    private static void writeContent(PackageEntryWriter writer, StringWriter stringWriter, List<Class<?>> views) {
+        List<Class<?>> processedClasses = views.stream().filter(writer::isProcessed).collect(Collectors.toList());
+        if (processedClasses.size() > 2) {
+            PluginRuntime.context().getExceptionHandler().handle(new InvalidSettingException(String.format(EXCEPTION_MESSAGE_TEMPLATE, writer.getXmlScope())));
```

Comment:
There's not quite a point in calling exception handler from a test routine because it has nothing to do with the Maven build workflow